### DISCUSSION TITLE
Make Llava to be configurable so that you can swap text model

### DISCUSSION THREAD

**Discussion by mergennachin** [pytorch/executorch#8191] [Ideas]
### 🚀 The feature, motivation and pitch

Current Llava uses Llama 2 7B as the pretrained text model. https://github.com/pytorch/executorch/blob/main/examples/models/llava/README.md

The [latest quantized Llama 1B/3B are good in terms of accuracy and size](https://github.com/pytorch/executorch/blob/main/examples/models/llama/README.md#llama-32-1b3b-and-quantized-1b3b-models). Let's make it swappable with these models.
